### DISCUSSION TITLE
Extreme lag in linux osu! lazer

### DISCUSSION THREAD

**Discussion by Linnuss** [ppy/osu#24454] [Q&A] [answered]
The game was running fine, more than 60 fps, but I don't know what happened, 2 months ago, the game is running at a stable 5 fps, I don't know if it was an update or what, but I want to play again and I don't know what else to do

**Comment by p00nmate** [comment]
#15188

**Reply by Linnuss** [reply, marked as answer]
Acabo de entrar al juego y después de estos dos meses se arregló por una actualización que acaba de llegar, gracias por tu ayuda igualmente.
I just entered the game and after these two months it was fixed by an update that just arrived, thanks for your help anyway.
